DCRack/dc_rack_server.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2011  Ted White
+# Copyright (C) 2011  Ted White 
```

Comment:
In general you just push more commits to the pull requested branch, but in this case see my notes below.